### PR TITLE
CDAP-11352 Reduce the log frequency of operational stats collection warnings

### PR DESCRIPTION
**Pull request by poornachandra**
JIRA - https://issues.cask.co/browse/CDAP-11352
Build - https://builds.cask.co/browse/CDAP-RUT1162-1

### CODE REVIEW

**Comment by sagarkapare** [comment]
LGTM 👍

File: cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/AbstractHDFSStats.java [hunk 1]
```diff
@@ -43,14 +43,17 @@
...
+import java.util.concurrent.TimeUnit;
 import javax.annotation.Nullable;
 
 /**
  * Base class for {@link OperationalStats} for HDFS
  */
 public abstract class AbstractHDFSStats extends AbstractOperationalStats {
   private static final Logger LOG = LoggerFactory.getLogger(AbstractHDFSStats.class);
-  private static final Logger READ_FAILURE_LOG = Loggers.sampling(LOG, LogSamplers.limitRate(60000));
+  private static final Logger READ_FAILURE_LOG =
```
Comment (bdmogal) [inline cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/AbstractHDFSStats.java:54]: Why do we have this rate different from the other one?

**Review by bdmogal — APPROVED** [review]
1 question.other than that, lgtm

File: cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/AbstractHDFSStats.java (hunk 1 above)
Comment (poornachandra) [reply cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/AbstractHDFSStats.java:54]: This one logs for all failures, not just HDFS and also it does not log the stack trace.

**Pull request merged**